src/connectors/radio.de.ts

```diff
@@ -0,0 +1,17 @@
+export {};
+
+Connector.playerSelector = 'div.player-nav-holder';
+
+Connector.artistTrackSelector = 'span';
```

Comment:
Do we need to assign this to the Connector prop? would it make sense to just pass `span` into `Util.getTextFromSelectors`?

Reply:
Right. Simplified it.